testing.sagebrush.master/Stepwat_Inputs/Input/bmassflags.in

```diff
@@ -53,6 +53,8 @@
 #       is reported immediately to the right of the group biomass.
 #       summary: average for the given year. Can only be
 #       output if GrpBmass is specified.
+# Grpwf == wild fires count across all the iterations.
```

Comment:
change to:
`# Grpwf == number of wildfires that occured across all iterations in each year`